docs/developers.rst

```diff
@@ -90,11 +90,6 @@ You may now locally run:
   pytest
```

Comment:
Note that all this should now also locally be doable with `tox`. Of course, this would only work on the platforms where everything works from `pip`, because that's what's needed for `tox` to work; so basically only under Linux, and assuming the non-pip dependencies are installed. I would add that option to these instructions, though, because if indeed you work on Linux, using tox locally makes you automatically consistent with the CI setup, which could ease development.